src/NHibernate/Impl/AbstractQueryImpl.cs

```diff
@@ -1048,5 +1050,8 @@ public override string ToString()
 		}
 
 		protected internal abstract IEnumerable<ITranslator> GetTranslators(ISessionImplementor sessionImplementor, QueryParameters queryParameters);
-	}
+
+		// Since v5.2
+		[Obsolete("This method has no usages and will be removed in a future version")]
+		protected internal abstract Task<IEnumerable<ITranslator>> GetTranslatorsAsync(ISessionImplementor sessionImplementor, QueryParameters queryParameters, CancellationToken cancellationToken);	}
```

Comment:
A mishap has occurred on the class closing brace. It is now on the method line instead of having its own.